package.json

```diff
@@ -51,7 +51,7 @@
     "terraformer-wkt-parser": "^1.1.2",
     "toposort-class": "^1.0.1",
     "uuid": "^3.2.1",
-    "validator": "^9.4.1",
+    "validator": "^10.4.0",
```

Comment:
Not sure if this is acceptable in a minor version bump, but looking at [validator's changelog](https://github.com/chriso/validator.js/blob/master/CHANGELOG.md#1000) I don't see any breaking changes in 10.0.

Reply:
Yes, I checked that there were no breaking changes that affected us